flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyClientServerSslTest.java

```diff
@@ -96,6 +108,7 @@ private void testValidSslConnection(Configuration sslConfig) throws Exception {
 		// session context is only be available after a session was setup -> this should be true after data was sent
 		SSLSessionContext sessionContext = sslHandler.engine().getSession().getSessionContext();
 		assertNotNull("bug in unit test setup: session context not available", sessionContext);
+		// TODO: sessionContext is from the client side which may not have a session cache at all (with openSSL it behaves that way)
```

Comment:
?

Reply:
done - by a little extra work here and there :(